routes/routes.js

```diff
@@ -15,8 +15,9 @@ router
 
 router.post("/locations", postLocation);
 
-router.patch("/locations/:location_id", (req, res) => {
-  res.send("Patch location");
-});
+// What is this for? It doesn't seem to do anything.
```

Comment:
I think this was a holder from the template we used to set things up. I think we were planning on patching dangerous , but this might already be used somewhere else